nixos/modules/virtualisation/linode-image.nix

```diff
@@ -0,0 +1,62 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.virtualisation.linodeImage;
+  defaultConfigFile = pkgs.writeText "configuration.nix" ''
+    _: {
+      imports = [
+        <nixpkgs/nixos/modules/virtualisation/linode-image.nix>
+      ];
+    }
+  '';
+in
+{
+  imports = [ ./linode-config.nix ];
+
+  options = {
+    virtualisation.linodeImage.diskSize = mkOption {
+      type = with types; either (enum (singleton "auto")) int;
```

Comment:
```suggestion
      type = with types; either (enum (singleton "auto")) uint;
```

Reply:
Hmm, was about to say I jumped the gun when fixing this before testing. Didn't think to until after, so I'll be doing it now. Looks like it doesn't recognize this as a valid unit.
![image](https://user-images.githubusercontent.com/17628961/185498463-db031bf8-5796-40f0-ada0-79580437ef4c.png)

Is this a type defined by nixpkgs? Might need a `lib.` proceeding it.

Reply:
Think I found something analogous with `types.ints.positive`. Testing that that works first before committing this time.

Reply:
Fixed in ae38e68